.github/workflows/publish-docs.yml

```diff
@@ -10,8 +10,10 @@ on:
       - 'main'
 
 env:
-  node-version: 16.x
-  python-version: 3.x
+  NODE_VERSION: 16.16.0 # renovate: datasource=node depName=node
+  PYTHON_VERSION: 3.10.5 # renovate: datasource=github-releases depName=python packageName=containerbase/python-prebuild
+  # Don't use PIPENV_VERSION:  https://github.com/pypa/pipenv/issues/3633#issuecomment-478250721
+  RENOVATE_PIPENV_VERSION: 2022.7.24 # renovate: datasource=pypi depName=pipenv
```

Comment:
Why are you pinning these environment variables? Will Renovate bot update those pinned versions for us?

Reply:
yes